src/main/java/kotlinx/reflect/lite/misc/JvmClassMapping.kt

```diff
@@ -0,0 +1,86 @@
+// Most logic copied from: https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/jvm/runtime/kotlin/jvm/JvmClassMapping.kt
+package kotlinx.reflect.lite.misc
```

Comment:
The declarations in this file are not planned for the initial release if I understand correctly, so we can make all of them `internal`. However, if we *do* want to publish them, then we should change the package name. `misc` is not very telling. `kotlinx.reflect.lite.jvm`, to mirror https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/java-primitive-type.html, would be better, for example.